docs/operations/api-reference.md

```diff
@@ -401,6 +401,30 @@ Returns total size and count for each interval within given isointerval.
 
 Returns total size and count for each datasource for each interval within given isointerval.
 
+#### Dynamic configuration
+
+See [Coordinator Dynamic Configuration](../configuration/index.md#dynamic-configuration) for details.
+
+Note that all _interval_ URL parameters are ISO 8601 strings delimited by a `_` instead of a `/`
+(e.g., 2016-06-27_2016-06-28).
+
+##### GET
+
+* `/druid/coordinator/v1/config`
+
+Retrieves current coordinator dynamic configuration.
+
+* `/druid/coordinator/v1/config/history?interval={interval}&count={count}`
+
+Retrieves history of changes to overlord dynamic configuration. Accepts `interval` and  `count` query string parameters
+to filter by interval and limit the number of results respectively.
+
+##### POST
+
+* `/druid/coordinator/v1/config`
+
+Update overlord dynamic worker configuration.
```

Comment:
Seems odd not to have the payload here, but I see that it is in the linked section.

Reply:
I'm indifferent on this note. Since other `POST` endpoints in this same doc do not include information on what the payload would look like, I think it is fine as is.

Reply:
Thanks @capistrant ! I think you are right, too. Everything API related in Druid could use an overhaul to be a little more user-friendly and that surely wasn't the goal of this PR.

Reply:
Ah yeah, I just copied this from the overlord section. It would be really nice to have example requests and responses for all of these API calls, but I think maybe that could be done as future work.